nestkernel/archiving_node.h

```diff
@@ -245,6 +245,126 @@ class Archiving_Node : public Node
   std::map< Name, SynapticElement > synaptic_elements_map_;
 };
 
+/**
+ * \class Clopath_Archiving_Node
+ * a archiving node which additionally archives parameters
+ * needed for the Clopath plasticity rule
+ */
+class Clopath_Archiving_Node : public Archiving_Node
```

Comment:
Moved to separate file for now. We will discuss your suggestion and have a look if this is possible.

Reply:
OK, but keep in mind that your argument rests on the assumption that one of you will keep working on these similar models. I understand your consideration in terms of workload but the danger of not creating a minimal implementation of a new archiving node _now_ is that future developers who are not you will miss your implementation and start from scratch, duplicating a lot of functionality.

Reply:
Yes, of course you are right. However, given our HBP task it seems reasonable to expect, that it will still be use who work on similar models in the near future ;)